packages/@aws-cdk/aws-redshift/lib/parameter-group.ts

```diff
@@ -22,6 +35,15 @@ abstract class ClusterParameterGroupBase extends Resource implements IClusterPar
    * The name of the parameter group
    */
   public abstract readonly clusterParameterGroupName: string;
+
+  /**
+   * Indicates if the parameter group parameters can be updated.
+   */
+  protected abstract updatableParameters: boolean;
```

Comment:
Can we add more explanation in this docstring? Why can this be false? Why can it be true? What conditions make it true vs false?